chrome/script.browserify.js

```diff
@@ -36,7 +36,7 @@ function view() {
     } else if (logins.length > 0) {
       results = logins.map(function(login) {
         let selector = "button.login";
-        let options = { onclick: getLoginData.bind(login) };
+        let options = { onclick: getLoginData.bind(login), title: "Fill form" + (localStorage.getItem("autoSubmit") ? " and submit" : "") };
```

Comment:
could you change the code to access localStorage only once and store result in the variable, before the `logins.map()`?

Also, this doesn't really work if you change autosubmit to "yes", save, and then change back to "no".
It should be instead `localStorage.getItem("autoSubmit") == "true"`